### PR TITLE
Add `scaledToFill` ViewModifier documentation and example

### PR DESCRIPTION
**Pull request by JaredDanielJones**
### Description

This pull request adds documentation and an example for the `scaledToFill` ViewModifier in SwiftUI. The following changes have been made:

1. Created a new folder: `content/swiftui/concepts/viewmodifier/terms/scaledToFill`
2. Added the `scaledToFill.md` file in the new folder, which contains an explanation of the `scaledToFill` ViewModifier and its usage.
3. Included an example image: `media/swiftui-viewmodifier-scaledToFill.png` to demonstrate the effect of the `scaledToFill` ViewModifier.

### Type of Change

- Adding a new entry

### Checklist

- [x] All writings are my own.
- [x] My entry follows the Codecademy Docs style guide.
- [x] My changes generate no new warnings.
- [x] I have performed a self-review of my own writing and code.
- [x] I have checked my entry and corrected any misspellings.
- [x] I have made corresponding changes to the documentation if needed.
- [x] I have confirmed my changes are not being pushed from my forked `main` branch.
- [x] I have confirmed that I'm pushing from a new branch named after the changes I'm making.
- [x] Under "Development" on the right, I have linked any issues that are relevant to this PR (write "Closes #<issue number> in the "Description" above).

### CODE REVIEW

**Review by yangc95 — APPROVED** [review]
@JaredDanielJones Awesome!

File: content/swiftui/concepts/viewmodifier/terms/scaledToFill/scaledToFill.md [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+
+struct MyStruct: View {
+    var body: some View {
+        Circle()
+            .scaledToFill()
+    }
+}
+```
+
+The image on the left shows the circle without the **`.scaledToFill()`** modifier applied. The image on the right shows the circle with the **`.scaledToFill()`** modifer applied.
```
Comment (SSwiniarski) [inline content/swiftui/concepts/viewmodifier/terms/scaledToFill/scaledToFill.md:41]:
```suggestion
The image on the left shows the circle without the `.scaledToFill()` modifier applied. The image on the right shows the circle with the `.scaledToFill()` modifier applied.
```

**Review by SSwiniarski — CHANGES_REQUESTED** [review]
@JaredDanielJones Good entry. I did a second review and only had one comment.

**Review by SSwiniarski — APPROVED** [review]
@JaredDanielJones Looks good.

**Comment by SSwiniarski** [comment]
@JaredDanielJones congrats on the new entry. Live link here: https://www.codecademy.com/resources/docs/swiftui/viewmodifier/scaledToFill

You're currently listed as an anonymous contributor. If you want to be credited follow the instructions below to add your GitHub username to your Codecademy profile. If not, there is nothing more you need to do.

image

To add your GitHub username to your Codecademy profile:

1.) Add your GitHub Username to your Codecademy Profile Details, you can get there on your desktop by pressing on your profile picture in the upper right-hand corner and selecting "Profile" in the drop-down box:

image

2.) Then log out and log back into Codecademy using your GitHub login:

image

**Note**: Your email used for Codecademy will have to be listed in your GitHub profile for the login feature to properly work. If you use the same email for both GitHub and Codecademy already, then you're set.

**Pull request merged**